gunrock/app/hits/hits_app.cuh

```diff
@@ -205,21 +201,21 @@ double gunrock_hits(
 }
 
 /*
- * @brief Simple interface take in graph as CSR format
+ * @brief Templated interface take in a graph in CSR format and return vertex hub and authority scores
  * @param[in]  num_nodes   Number of veritces in the input graph
  * @param[in]  num_edges   Number of edges in the input graph
  * @param[in]  row_offsets CSR-formatted graph input row offsets
  * @param[in]  col_indices CSR-formatted graph input column indices
- * @param[in]  edge_values CSR-formatted graph input edge weights
- * @param[in]  num_runs    Number of runs to perform SM
- * @param[out] subgraphs   Return number of subgraphs
- * \return     double      Return accumulated elapsed times for all runs
+ * @param[in]  num_iter    Number of iterations to perform HITS
+ * @param[out] hub_ranks   Vertex hub scores
+ * @param[out] auth ranks  Vertex authority scores
+ * \return     double      Return accumulated elapsed times for all iterations
```

Comment:
Here you mentioned returning accumulated elapsed times for all iterations but you only run one.